it/src/main/java/com/google/cloud/teleport/it/datastream/DatastreamResourceManager.java

```diff
@@ -0,0 +1,30 @@
+
+package com.google.cloud.teleport.it.datastream;
+
+import com.google.cloud.datastream.v1.ConnectionProfile;
+import com.google.cloud.datastream.v1.Stream;
+
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Interface for managing datastream resources in integration tests.
+ */
+public interface DatastreamResourceManager {
```

Comment:
This should extend `ResourceManager`
```suggestion
public interface DatastreamResourceManager extends ResourceManager {
```

Reply:
done